### PR TITLE
Add FusionAuth SAML integration guide

### PR DESCRIPTION
**Pull request by vcampitelli**
Following [Aiven guidelines](https://github.com/aiven/devportal/blob/main/CONTRIBUTING.rst#howto-article-template) to create an article demonstrating how to use [FusionAuth](https://fusionauth.io) as the IdP via SAML.

### CODE REVIEW

**Comment by vcampitelli** [comment]
Hi there, I've rebased the branch and added some missing words to vale's dict. Thanks for the heads up!

File: docs/platform/howto/saml/setup-saml-fusionauth.rst [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+* ``SAML Entity ID``: the **Entity Id** from FusionAuth.
+* ``SAML Certificate``: paste the certificate you’ve copied from the ``Generating certificate`` two steps above.
+
+5. Click **Edit method** to save your changes.
+
+6. Toggle on **Enable authentication method** at the top of the page.
+
+You can use the **Signup URL** to invite new users, or the **Account link URL** for those that already have an Aiven user account.
+
+Testing
```
Comment (staceysalamon-aiven) [inline docs/platform/howto/saml/setup-saml-fusionauth.rst:107]:
```suggestion
```

File: docs/platform/howto/saml/setup-saml-fusionauth.rst [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+-------
+
+Open the Signup URL you copied above in an incognito tab or using another browser. You’ll reach the signup page below.
```
Comment (staceysalamon-aiven) [inline docs/platform/howto/saml/setup-saml-fusionauth.rst:110]:
```suggestion
```

File: docs/platform/howto/saml/setup-saml-fusionauth.rst [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+.. image:: /images/platform/howto/saml/fusionauth/login-sso.png
```
Comment (staceysalamon-aiven) [inline docs/platform/howto/saml/setup-saml-fusionauth.rst:112]:
```suggestion
```

File: docs/platform/howto/saml/setup-saml-fusionauth.rst [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+   :alt: Logging in to Aiven.
```
Comment (staceysalamon-aiven) [inline docs/platform/howto/saml/setup-saml-fusionauth.rst:113]: ```suggestion```

File: docs/platform/howto/saml/setup-saml-fusionauth.rst [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+Click on **Sign up with FusionAuth**, which will redirect you to the FusionAuth login page. Fill in your credentials and submit the form to be taken back to the Aiven console, already logged in and part of the ``Developers`` team.
```
Comment (staceysalamon-aiven) [inline docs/platform/howto/saml/setup-saml-fusionauth.rst:115]:
```suggestion
```

File: docs/platform/howto/saml/setup-saml-fusionauth.rst [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+* ``SAML Certificate``: paste the certificate you’ve copied from the ``Generating certificate`` two steps above.
+
+5. Click **Edit method** to save your changes.
+
+6. Toggle on **Enable authentication method** at the top of the page.
+
+You can use the **Signup URL** to invite new users, or the **Account link URL** for those that already have an Aiven user account.
+
+Testing
+-------
```
Comment (staceysalamon-aiven) [inline docs/platform/howto/saml/setup-saml-fusionauth.rst:108]:
```suggestion
```

File: docs/platform/howto/saml/setup-saml-fusionauth.rst (hunk 1 above)
Comment (staceysalamon-aiven) [reply docs/platform/howto/saml/setup-saml-fusionauth.rst:107]: I don't think the testing section or the screenshot is needed here.

**Review by staceysalamon-aiven — CHANGES_REQUESTED** [review]
Thank you @vcampitelli for writing up these instructions! I've made suggestions based on our style guide and to more closely match the other docs. Any questions, let me know!

File: docs/platform/howto/saml/setup-saml-fusionauth.rst [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+
+.. image:: /images/platform/howto/saml/fusionauth/create-api-key.png
+   :alt: Creating API Key.
+
+After being redirected back to the API Keys page, click on the ``Key`` for the created item to reveal its actual value and copy it. You’ll have to provide this value to the script soon.
+
+.. image:: /images/platform/howto/saml/fusionauth/grab-api-key.png
+   :alt: Grabbing API Key.
+
+Now that your application is ready, clone `this GitHub repository <https://github.com/FusionAuth/fusionauth-example-scripts>`__ and execute the script located in ``rsa-certificate``.
```
[the remark this answers is not in the document]
Comment (vcampitelli) [reply docs/platform/howto/saml/setup-saml-fusionauth.rst:71]: Yes, I've renamed it and forgot to change it here. Thanks!

**Comment by vcampitelli** [comment]
Hi @staceysalamon-aiven , thank you for reviewing this! I've applied your suggestions and made the other changes. Please let me know if there is anything else!

**Comment by staceysalamon-aiven** [comment]
> Hi @staceysalamon-aiven , thank you for reviewing this! I've applied your suggestions and made the other changes. Please let me know if there is anything else!

Thanks for doing that! There's an error on line 59 that I think is due to the following lines not being indented?

**Comment by vcampitelli** [comment]
Hmm that was strange because it compiled well with `make livehtml`.. But it's fixed! Thanks!

**Review by staceysalamon-aiven — APPROVED** [review]
Looks good! Thanks again for your contribution!

**Pull request merged**